### ISSUE TITLE
Latest docker breaks sightglass

### ISSUE THREAD

**Issue by peterhuene** [bytecodealliance/sightglass#116]
It appears that `docker build` for docker version `Docker version 20.10.5, build 55c4c88` (macOS) outputs this as the last line:

```text
Use 'docker scan' to run Snyk tests against images to find vulnerabilities and learn how to fix them
```

This breaks sightglass such that it parses `them` as the image id for what was just build, causing this error message:

```text
Unable to find image 'them:latest' locally
Error response from daemon: pull access denied for them, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
 ERROR sightglass_artifact::docker > Failed: Child { stdin: None, stdout: None, stderr: None }
Error: failed to execute docker command:
```

**Comment by peterhuene** [comment]
I seem to be building with BuildKit enabled by default, which also breaks sightglass as the output goes to stderr and not stdout (only the `docker scan` message goes to stdout.

This problem persists, though, with `DOCKER_BUILDKIT=0` as the `docker scan` message still gets printed.

**Comment by abrown** [comment]
Yeah, [docker.rs](https://github.com/bytecodealliance/sightglass/blob/main/crates/artifact/src/docker.rs) was designed as a placeholder using CLI commands until I could get time to use the Docker API properly. I believe I tried out [bollard](https://crates.io/crates/bollard) but stopped because I ran into issues; the issue may have been that I thought the `async` was spreading too much or that I couldn't get it to work with `podman` (on Fedora). But this bug is an important one to fix. (I don't know what workaround to suggest in the meantime...).

**Comment by abrown** [comment]
Actually, the workaround is likely to just build everything natively and avoid Docker completely: the bench-api shared library in the Wasmtime repository can be built manually and the path passed on the command line. Likewise, any benchmark-building can be done manually as well and the Wasm file path passed on the command line.

**Comment by peterhuene** [comment]
The workaround of manually building the engine and passing `-e <path>` to the benchmark command does work for me 👍

**Comment by abrown** [comment]
This issue should be closed now that #170 and #177 are merged.
